exporter/awsxrayexporter/awsxray.go

```diff
@@ -33,16 +33,18 @@ const (
 	maxSegmentsPerPut = int(50) // limit imposed by PutTraceSegments API
 )
 
-// NewTraceExporter creates an component.TraceExporterOld that converts to an X-Ray PutTraceSegments
+// NewTraceExporter creates an component.TraceExporter that converts to an X-Ray PutTraceSegments
 // request and then posts the request to the configured region's X-Ray endpoint.
-func NewTraceExporter(config configmodels.Exporter, logger *zap.Logger, cn connAttr) (component.TracesExporter, error) {
+func NewTraceExporter(
```

Comment:
nit: Does this need to be public?

Reply:
Looks like there are a few others also that are public but could be private - I addressed the two you noticed here, let me followup with more cleanup.